packages/modules/manage-versions/src/components/ManageVersions/ChangesTab/ChangesTab.tsx

```diff
@@ -62,6 +61,19 @@ const ChangesTab = (props: ChangesTabProps) => {
             Header: stringsOverrides.description,
             accessor: "description",
           },
+          {
+            id: "CHANGED_FILES",
+            Header: stringsOverrides.changedFiles,
+            Cell: (props: CellProps<Changeset>) => {
+              const changedFiles =
+                props.row.original.synchronizationInfo.changedFiles;
+              return (
+                <span>
```

Comment:
I don't think this span wrapper is needed.

Reply:
change pushed